src/CustomElementInternalSymbols.js

```diff
@@ -0,0 +1,7 @@
+const randomString = Math.random().toString(32).substring(2);
+
+export const state = '_CE_state_' + randomString;
+export const definition = '_CE_definition_' + randomString;
+export const shadowRoot = '_CE_shadowRoot_' + randomString;
+export const documentLoadHandled = '_CE_documentLoadHandled_' + randomString;
+export const patched = '_CE_patched_' + randomString;
```

Comment:
This should be fixed now. I added externs for these names so that closure doesn't mangle them and I just reference them directly, rather than with `[]`.